content/en/docs/tasks/extend-kubernetes/socks5-proxy-access-api.md

```diff
@@ -0,0 +1,148 @@
+{{</ note >}}
+
+Figure 1 represents what you're going to achieve in this task.
+
+* You have a client compute from where you're going to create requests to talk to the Kubernetes API
+* The Kubernetes server/API is hosted on a remote server.
+* You will use SSH client and server software to create a secure SOCKS5 tunnel between the local and
+  the remote server. The HTTPS traffic between the client and the Kubernetes API will flow over the SOCKS5
+  tunnel, which is itself tunnelled over SSH.
+
+{{< mermaid >}}
+graph LR;
+
+  subgraph local[Local client machine]
+  client([client])-- local <br> traffic .->  local_ssh[Local SSH <br> SOCKS5 proxy];
+  end
+  local_ssh[SSH <br>SOCKS5 <br> proxy]-- SSH Tunnel -->sshd
+  
+  subgraph remote[Remote server]
+  sshd[SSH <br> server]-- local traffic -->service1;
+  end
+  client([client])-. proxied HTTPs traffic <br> going through the proxy .->service1[Kubernetes API];
+
+  classDef plain fill:#ddd,stroke:#fff,stroke-width:4px,color:#000;
+  classDef k8s fill:#326ce5,stroke:#fff,stroke-width:4px,color:#fff;
+  classDef cluster fill:#fff,stroke:#bbb,stroke-width:2px,color:#326ce5;
+  class ingress,service1,service2,pod1,pod2,pod3,pod4 k8s;
+  class client plain;
+  class cluster cluster;
+{{</ mermaid >}}
+Figure 1. SOCKS5 tutorial components
+
+## Using ssh to create a SOCKS5 proxy
+
+This command starts a SOCKS5 proxy between your client machine and the intermediate
+host. The SOCKS5 proxy lets you connect via that immediate host to your cluster's API
+server (provided that the traffic is permitted).
+
+```shell
+# The SSH tunnel continues running in the foreground after you run this
+ssh -D 1080 -q -N username@kubernetes-jump-box.example
+```
+
+* `-D 1080`: opens a SOCKS proxy on local port :1080.
+* `-q`: quiet mode. Causes most warning and diagnostic messages to be suppressed.
+* `-N`: Do not execute a remote command. Useful for just forwarding ports.
+* `username@kubernetes-jump-box.example`: the remote SSH server where the Kubernetes cluster is running.
+
+## Client configuration
+
+To explore the Kubernetes API you'll first need to instruct your clients to send their queries through
+the SOCKS5 proxy we created earlier.
+
+For command-line tools, set the `https_proxy` environment variable and pass it to commands that you run.
+
+```shell
+export https_proxy=socks5h://localhost:1080
+```
+
+When you set the `https_proxy` variable, tools such as `curl` route HTTPS traffic through the proxy
+you configured. For this to work, the tool must support SOCKS5 proxying.
+
+{{< note >}}
+In the URL https://localhost/api, `localhost` does not refer to your local client computer.
+Instead, it refers to the the endpoint that the intermediate host (“jump box”) knows as `localhost`.
+The `curl` tool sends the hostname from the HTTPS URL over SOCKS, and the intermediate
+host resolves that locally (to an address that belongs to its loopback interface).
```

Comment:
at the beginning you say

> To follow this task you need an intermediate host (also known as a “jump box”). The task assumes that
the intermediate host is separate from your Kubernetes cluster.

but here it assumes the apiserver is running on the jump box, since you connect to localhost of the jumpbox, or am I getting it wrong?

Reply:
@aojea ohhh thank you so much for catching that! You're 100% right. That's what I meant initially, to keep it simple the ssh-server **is** running on the same machine as the kubernetes server.
It doesn't mean that it can't run in a dedicated remote machine (via a jump-box) but this is outside the scope of this tutorial.

I removed this notion of "jump-box" that was introduced during the review process and reworked slightly the phrasing on the document

ptal :)